src/Model/Process.php

```
@@ -305,13 +315,19 @@ public static function checkInactive($id) {
 	 */
 	public static function deleteInactive()
 	{
-		if (is_null(self::$instance) || is_null(self::$instance->parent)) {
+		if (is_null(self::$instance)) {
```

And maybe again a silent failure?